pkg/sql/execinfrapb/data.proto

```diff
@@ -394,5 +394,10 @@ message TracingAggregatorEvents {
   optional bytes flow_id = 2 [(gogoproto.nullable) = false,
     (gogoproto.customname) = "FlowID",
     (gogoproto.customtype) = "FlowID"];
+  // Identifier of this component, within the domain of components of the same
```

Comment:
too many ids :)
the comment is a bit hard (for me) to understand - what is a component?

Reply:
can it just be processor id?

Reply:
thanks for the chat, got it.